### DISCUSSION TITLE
Inactive/deactived Sensor Smoke/Presence

### DISCUSSION THREAD

**Discussion by prawnhitter** [SukramJ/aiohomematic#1534] [General]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic) right now
- [X] I have read the [FAQs](https://github.com/danielperna84/custom_homematic#frequently-asked-questions) right now
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] I am aware that an integration that was installed via HACS also needs to be updated via HACS
- [X] The Backend (CCU/Homegear/...) is working as expected, devices are taught in and are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Hi, I have discovered that my Smokesensor HmIP-SWSD / Pesencedetector HmIP-SPI can not be activated or deactivated.

In the CCU interface I can use them as normal as usual. I can activate the sensor when I leave and deactivate when I come back. But in Homeassistant the Device is there but the sensor is in status „deactivated“ even though I can not change it there.
![IMG_9543](https://github.com/danielperna84/hahomematic/assets/50422850/137b8c55-f86b-465d-8e8a-4cd62f2ea0a1)
![IMG_9542](https://github.com/danielperna84/hahomematic/assets/50422850/64686eb6-d9de-4d73-8352-74d0469d3d9f)

In the first Versions was it possible, now there seams a connection/interpreter error. Can you verify this?

### What version of HomematicIP (local) has the issue?

1.60.1

### What was the last working version of HomematicIP (local)?

1.60.1

### What type of installation are you running?

Home Assistant Container

### What type of installation are you running for your homematic backend?

CCU3

### Which version of your homematic backend are you running?

3.75.7

### Which config details do you use

- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

### Log file extract. Anything in the logs that might be useful for us? The log (Setting/System/Logs -> load full log) is the best source to support trouble shooting!

```txt
-
```

### Additional information

-

**Comment by SukramJ** [comment]
Please add the log file:
> Log file extract. Anything in the logs that might be useful for us? The log (Setting/System/Logs -> load full log) is the best source to support trouble shooting!

**Comment by SukramJ** [comment]
Basically this is not a problem with the integration, as these devices work without a problem in my environment, So I think it‘s a communication problem on your system. 
The logs might have more information.

> In the first Versions was it possible

This information is useless, unless you specify a version!

**Comment by prawnhitter** [comment]
Hi SukramJ,

Thanks for your response. Maybe this information seems useless, but it also means that this was working some time ago and may have to do something with the environment.
All other devices are working fine in my environment, only those two having a topic with the sensor. Maybe we can eleborate together a solution here.
This is a pic from CCU3 where i can change the status.
![image](https://github.com/danielperna84/hahomematic/assets/50422850/632a61a1-bb36-49e2-8b5b-b937590bbc1f)

[home-assistant_2024-04-28T18-39-54.781Z.log](https://github.com/danielperna84/hahomematic/files/15143230/home-assistant_2024-04-28T18-39-54.781Z.log)
Also please find attached the logfile you requested.

Thank you for your help

**Comment by SukramJ** [comment]
Your logs are full of errors that indicate communication problems from HA to CCU, and maybe from CCU to HA.
Please check the [docs](https://github.com/danielperna84/custom_homematic?tab=readme-ov-file#homematicip-local-documentation) especially for docker related infos (search for docker).

**Comment by prawnhitter** [comment]
Hi, okay I will check that again and get back to you in the next two days.

**Comment by SukramJ** [comment]
I move this to discussion, because I see no issue with the integration.
